thunder/tests/distributed/test_dtensor.py

```diff
@@ -161,6 +161,55 @@ def fn(x, w):
         with pytest.raises(RuntimeError, match="has changed for cotangent between tracing and runtime"):
             torch.autograd.grad(actual, (in_dtensor, w_dtensor), g_o)
 
+    @common_utils.parametrize("executor", tuple(executors_map.keys()))
+    def test_dtensor_convert_element_type(self, executor):
+        from thunder.torch.experimental.dtensor_torch_and_prims import dtensor_convert_element_type_prim
+        from thunder.core.dtypes import bfloat16
```

Comment:
now most environments would be compatible with bfloat16 but how about using https://github.com/Lightning-AI/lightning-thunder/blob/68b4c4cbba9be58c9b3404a7847b2e1fa8256cd4/thunder/tests/bf16.py#L5?

Would this be a bit too paranoid/meticulous?

Reply:
Right, it should be safe. Have used `dtypes.bfloat16` where `dtypes` is imported at module level. Thanks!